okhttp/src/main/java/okhttp3/internal/Util.java

```diff
@@ -642,4 +647,20 @@ private static String inet6AddressToAscii(byte[] address) {
     }
     return result.readUtf8();
   }
+
+  public static X509TrustManager platformTrustManager() {
+    try {
+      TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(
+          TrustManagerFactory.getDefaultAlgorithm());
+      trustManagerFactory.init((KeyStore) null);
+      TrustManager[] trustManagers = trustManagerFactory.getTrustManagers();
+      if (trustManagers.length != 1 || !(trustManagers[0] instanceof X509TrustManager)) {
```

Comment:
Curious about this common approach vs first compatible of Netty

https://github.com/netty/netty/blame/a137291ad161c970259d9a5d7cc206c026f08ee7/handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java#L504-L510